README.md

```diff
@@ -1,2 +1,150 @@
-# module-ballerinax-dayforce
-The Ballerina Ceridian Dayforce connector provides a client to interact with the Ceridian Dayforce APIs.
+# Ballerina Dayforce Connector
+
+[![Build Status](https://github.com/ballerina-platform/module-ballerinax-dayforce/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-dayforce/actions?query=workflow%3ACI)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-dayforce/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-dayforce)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-dayforce.svg)](https://github.com/ballerina-platform/module-ballerinax-dayforce./commits/master)
+[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-dayforce/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-dayforce/actions/workflows/build-with-bal-test-native.yml)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+## Overview
+
+Dayforce is a comprehensive human capital management system that covers the entire employee lifecycle including HR, payroll, benefits, talent management, workforce management, and services. The entire system resides on cloud that takes the burden of managing and replicating data on-premise.
+
+The Dayforce connector allows you to access the REST API of Dayforce HCM. 
+
+This connector has been tested exclusively in the Dayforce developer sample environment. If you encounter any technical issues, please reach out to us using the links provided in the [Useful Links](#useful-links) section.
+
+## Setup guide
+
+### Step 1: Create a Dayforce account
+
+Navigate to the [Dayforce](https://developers.dayforce.com/Build/Home.aspx) website and register. Follow the instructions to create an account. If your company has already purchased a namespace in Dayforce, use that to sign up. Otherwise, you can still use their `sample` environment by selecting `sample` option.
+
+### Step 2: Obtain the user credentials to access the Dayforce API
+
+To get the credentials for the `sample` environment, follow the instructions below:
+1. Navigate to API Explorer > Employee > GET Employees.
+2. Locate the basic authentication credentials, including the `username` and `password`, for the sample environment.
+
```

Comment:
Shall we add include screenshots to indicate the mentioned steps, wherever possible (similar to what we have done in other connectors)?

Reply:
Fixed in [f54eb34](https://github.com/ballerina-platform/module-ballerinax-dayforce/pull/1/commits/f54eb34eb38de4c0777362f209ecad3871052ee2)